### PR TITLE
[V3 Info] Don't rely on redirect

### PR DESCRIPTION
**Pull request by TheWyn**
### Type

- [X] Bugfix
- [ ] Enhancement
- [ ] New feature

### Description of the changes

Fixes following trace:

```
Exception in command 'info'
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/discord/ext/commands/core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/redbot/core/core_commands.py", line 76, in info
    data = await r.json()
  File "/usr/lib/python3.6/site-packages/aiohttp/client_reqrep.py", line 730, in json
    headers=self.headers)
aiohttp.client_exceptions.ClientResponseError: 0, message='Attempt to decode JSON with unexpected mimetype: text/plain; charset=utf-8'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/discord/ext/commands/bot.py", line 886, in invoke
    yield from ctx.command.invoke(ctx)
  File "/usr/lib/python3.6/site-packages/discord/ext/commands/core.py", line 514, in invoke
    yield from injected(*ctx.args, **ctx.kwargs)
  File "/usr/lib/python3.6/site-packages/discord/ext/commands/core.py", line 71, in wrapped
    raise CommandInvokeError(e) from e
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: ClientResponseError: 0, message='Attempt to decode JSON with unexpected mimetype: text/plain; charset=utf-8'
```

### CODE REVIEW

**Comment by Redjumpman** [comment]
Just started experiencing this problem myself. The above changes fixes it for me.

**Comment by Bakersbakebread** [comment]
`Starting alfred...
Connected to Discord. Getting ready...
Loading packages...
[01/05/2018 12:26] ERROR events on_error 174: Exception in on_ready
Traceback (most recent call last):
  File "d:\alfred\venv\lib\site-packages\discord\client.py", line 224, in _run_event
    yield from coro(*args, **kwargs)
  File "d:\alfred\venv\lib\site-packages\redbot\core\events.py", line 114, in on_ready
    data = await r.json()
  File "d:\alfred\venv\lib\site-packages\aiohttp\client_reqrep.py", line 730, in json
    headers=self.headers)
aiohttp.client_exceptions.ClientResponseError: 0, message='Attempt to decode JSON with unexpected mimetype: text/plain; charset=utf-8'`

Getting the same error, above fixes for me.

**Comment by bobloy** [comment]
Bread, your error is not the same error. Your error is in events.py, not core_commands.py

File: redbot/core/core_commands.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def __init__(self, bot):
         owner = app_info.owner
 
         async with aiohttp.ClientSession() as session:
-            async with session.get("http://pypi.python.org/pypi/red-discordbot/json") as r:
+            async with session.get("https://pypi.python.org/pypi/red-discordbot/json") as r:
```
Comment (bobloy) [inline redbot/core/core_commands.py]: Replace this with `session.get('{}/json'.format(red_pypi))` defined above

**Review by bobloy — CHANGES_REQUESTED** [review]
`core/events.py` has the same problem and should be updated as well.

**Review by Kowlin — APPROVED** [review]

**Pull request merged**